pkg/ddl/partition.go

```diff
@@ -3307,31 +3322,21 @@ func (w *worker) onReorganizePartition(jobCtx *jobContext, job *model.Job) (ver
 			// In the next step, StateDeleteOnly, wait to verify the TiFlash replicas are OK
 		}
 
-		bundles, err := alterTablePartitionBundles(metaMut, tblInfo, tblInfo.Partition.AddingDefinitions)
+		changed, err := alterTablePartitionBundles(metaMut, tblInfo, tblInfo.Partition.AddingDefinitions)
 		if err != nil {
 			if !changesMade {
 				job.State = model.JobStateCancelled
 				return ver, errors.Trace(err)
 			}
 			return rollbackReorganizePartitionWithErr(jobCtx, job, err)
 		}
-
-		if len(bundles) > 0 {
-			if err = infosync.PutRuleBundlesWithDefaultRetry(context.TODO(), bundles); err != nil {
-				if !changesMade {
-					job.State = model.JobStateCancelled
-					return ver, errors.Wrapf(err, "failed to notify PD the placement rules")
-				}
-				return rollbackReorganizePartitionWithErr(jobCtx, job, err)
-			}
-			changesMade = true
-		}
+		changesMade = changesMade || changed
```

Comment:
Seems this method is using `rollbackLikeDropPartition` to rollback a DDL. `rollbackLikeDropPartition` does not rollback the group with ID `TiDB_DDL_{newTableID}`

Reply:
newTableID will be added to the `TablePartitionArgs.OldPhysicalTableIDs` along with the other new rolled backed partitions in `rollbackLikeDropPartition`, so that it will later on be picked up by GC to both clean up old rows as well as the Table and Partition Bundles.